From 1735aab1c475e66b079059cb00f7ebf46916eca1 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 22 Jun 2022 12:14:47 +0300
Subject: [PATCH] [SW-0] boot_desc: move arcboot header before fw if

fw_if header has packed structures, the definition of __packed
is in arcboot header. so the arcboot header should be included
before the fw_if header.

Change-Id: I893c77bb18cbd3676200e05da5e0fad444cc6c88
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/common/boot_desc.c b/common/boot_desc.c
index 092819d..a0cb8ee 100644
--- a/common/boot_desc.c
+++ b/common/boot_desc.c
@@ -7,10 +7,10 @@
 #include <stdint.h>
 #include <string.h>
 #include <stdbool.h>
+#include <arcboot.h>
 #include <hl_fw_if.h>
 #include "hl_boot_desc_rom.h"
 #include <common.h>
-#include <arcboot.h>
 #include "boot_desc.h"
 #include "dma.h"
 #include "spi_flash.h"
